etc/httpd/global.conf

```diff
@@ -875,10 +876,18 @@ RewriteRule ^/media/flash/playerWithControls.swf - [F]
 # Bug 896474
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?about(/?|/.+)$ /b/$1about$2 [PT]
 
-# Bug 987852
-RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?MPL(/?|/.+)$ /b/$1MPL$2 [PT]
-RewriteRule ^/MPL/1.1/index\.txt /media/MPL/1.1/index.txt [L,PT]
+# Bug 987852 & 1201914
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?MPL/1.1/FAQ.html /$1/MPL/1.1/FAQ/ [L,PT]
```

Comment:
> These ".html" urls should be redirects, not pass-throughs. Are there no longer any URLs with ".html" at all? Alternately are these the full list? If it is, it looks like we can pretty much just do a single rule for this:

Good idea. I've updated with this single rule. The only other `.html` urls are redirects to the website-archive that happen higher up in the file.

I left it as a separate commit, but will squash once we're all set.